pokemongo_bot/stepper.py

```diff
@@ -68,7 +68,7 @@ def _walk_to(self, speed, lat, lng, alt):
         steps = (dist + 0.0) / (speed + 0.0)  # may be rational number
         intSteps = int(steps)
         residuum = steps - intSteps
-        logger.log('[#] Walking from ' + str((i2f(self.api._position_lat), i2f(
+        logger.log('[#] Walking ' + str(ceil(dist)) + 'M from ' + str((i2f(self.api._position_lat), i2f(
```

Comment:
Can you use the format_dist function from utils? That way it will be in whatever unit the user has configured.